docs/openDAQ-streaming.md

```diff
@@ -158,8 +158,11 @@ This allows the client to know when all signals are described and data is to be
 ### Value Index
 An uint64 sent with each value of an implicit signal. It is used to align the value of the implicit signal with the table steps.
 
+### Related Signals
+A signal may have related signals that carry additional information. Exemples are domain signal and status signal.
+Each relation is described by the id of the related signal and the type of the related signal (i.e. "domain", "status").
 
-
+Only signal that are already subscribed may be used as related signal!

```

Comment:
signal -> signals both places.
I thought we auto subscribed to related signals?